pom.xml

```diff
@@ -4,7 +4,8 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>2.17</version>
+    <version>2.33</version>
```

Comment:
👍

Reply:
I would take advantage this PR for updating the parameter `<url>`for using `https`

Reply:
Will do.